webanno-model/src/main/java/de/tudarmstadt/ukp/clarin/webanno/model/AnnotationLayer.java

```diff
@@ -438,4 +440,14 @@ public void setReadonly(boolean aReadonly)
     {
         readonly = aReadonly;
     }
+    
```

Comment:
Mind that all changes that are made here must be made to `de.tudarmstadt.ukp.clarin.webanno.export.model.AnnotationLayer` as well and the import/export code must be updated correspondingly.